static/js/components/email/EmailCompositionDialog_test.js

```diff
@@ -166,14 +166,22 @@ describe('EmailCompositionDialog', () => {
     renderDialog({
       filters: [{
         id: '1',
-        name: "key",
-        value: "test"
+        name: "program.enrollments.semester",
+        value: "2015"
+      }, {
+        id: '2',
+        name: "ES",
+        value: "foo"
       }]
     }, { renderRecipients: SEARCH_RESULT_EMAIL_CONFIG.renderRecipients });
 
     assert.include(
-      getDialog().querySelector('.sk-selected-filters-option__name').textContent,
-      "key: test"
+      getDialog().querySelector('.sk-selected-filters').textContent,
+      "Semester: 2015"
+    );
+    assert.include(
+      getDialog().querySelector('.sk-selected-filters').textContent,
+      "Spain: foo"
```

Comment:
this logic is also present in `static/js/components/search/ModifiedSelectedFilter.js`, 
which is code to use in displaying facets.